_data/faq.yml

```diff
@@ -0,0 +1,38 @@
+spn:
+  - q: Is the SPN like Tor?
+    a: The SPN is heavily inspired by Tor, but the mission is different. Next to tightly integrating with other privacy modules, the main focus of the SPN is easy protection for your whole computer, not just your browser. Additionally it spreads your connections across the globe instead of routing everything through the same circuit.
+    link:
+      text: Read our comparison blog to find out more.
+      url: https://safing.io/blog/2020/01/22/how-the-spn-compares-to-tor/
+  - q: How does the SPN compare to VPNs?
+    a: The VPN technology was built to securely connect two company locations. However, the SPN was developed from the ground up to protect user privacy. As a result the SPN can easily outperform VPNs in every aspect.
+    link:
+      text: Read our blog post to read all about the differences.
+      url: /blog/2020/01/22/how-the-spn-compares-to-vpns/
+  - q: Are all connections routed through the SPN?
+    a: Yes, when enabled the Portmaster routes all Internet connections through the Safing Privacy Network.
+  - q: Does the SPN route all connections individually? So every connection exits on a different node?
+    a: Yes. Every connection is routed separately, giving you a vast amount of identities (IP addresses).
+    link:
+      text: View this explainer image showing a computer connecting to Steam, ProtonMail and Spotify.
+      url: https://safing.io/assets/img/spn-labeled/spn-community.png
+  - q: Does the SPN route connections on a per app basis?
+    a: No, it routes every connection individually. So as an example, when you open several websites in different tabs in your browser, every connection will be calculated individually.
+  - q: Am I totally safe from tracking with the SPN? What do service providers see when connecting through the SPN?
+    a: No. This is just one piece of the puzzle. The SPN only protects your IP-address from exposure, but there are still many other ways services can track and identify you. These include fingerprinting (mostly in browsers), authentication (when logged in to a service), mouse movements (malicious javascripts), etc...
+  - q: Can services still identify me through other methods?
+    a: Yes, this is just one piece of the puzzle. There are multiple ways services track you, like browser fingerprinting, malicious javascripts, etc... And if you log into services, such as your Google account, they will mostly still know it is you, even though you protect your IP address.
```

Comment:
Accurate, but somewhat a duplicate.
Suggestions only.

Reply:
Yeah, we also have this set of questions on the SPN right? I think when you're a user you first read the question, and if it interests you, you read on. I want to tackle multiple people, and it usually does not hurt saying the same thing twice or thrice